Yoast/wordpress-seo
# Moves redirects to integration

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Enable attachment redirect under search appearance - media
* Visit an attachment page and see the page being redirected to the image.
* Disable the author archives (under search appearance - archives) and verify the author page is redirected to the home.
* Disable the date archives (under search appearance - archives and verify a date archive page is redirected to the home.
* Create a post and remember it's id. Add the following code somewhere in the code base:
```
add_action( 'init', function() {
   \WPSEO_Meta::set_value( 'redirect',  'https://hyves.nl', 'the post id you remembered' );
} );
```
* Visit the post page and make sure you are redirected to the domain you set in the code above.

## UI changes

## Documentation

## Quality assurance


Fixes #
